### PR TITLE
Add machine learning courses in many universities

### PR DESCRIPTION
**Pull request by tuanphuc**
There are many resources (videos, lecture notes, problem sets ...) provided by many courses in Universities like Stanford, Cambridge or Oxford that can be useful to explore

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -302,6 +302,11 @@ Each day I take one subject from the list below, read it cover to cover, take no
 - [DigitalMind's Artificial Intelligence resources](http://blog.digitalmind.io/post/artificial-intelligence-resources)
 - [Awesome Machine Learning](https://github.com/josephmisiti/awesome-machine-learning)
 - [CreativeAi's Machine Learning](http://www.creativeai.net/?cat%5B0%5D=machine-learning)
+- [ ] Machine Learning courses in Universities
+    - [Stanford](http://ai.stanford.edu/courses/)
+    - [Machine Learning Summer Schools](http://mlss.cc/)
+    - [Oxford](https://www.cs.ox.ac.uk/people/nando.defreitas/machinelearning/)
+    - [Cambridge](http://mlg.eng.cam.ac.uk/)
```
Comment (ZuzooVn) [inline README.md]: We already had Machine Learning Summer Schools in the "Resource" section.
I think we should put [ ] before each university and move "Machine Learning courses in Universities" above "Flipboard Topics"

Comment (tuanphuc) [inline README.md]: Agree ! I thick that we can remove the "Machine Learning Summer Schools" (that is already there) after making all the changes that you mentioned. Thanks

**Comment by ZuzooVn** [comment]
LGTM, Thanks

**Comment by TrinhDinhPhuc** [comment]
Thank so much!

Vào 10-11-2016 18:47, "Nam Vu" <notifications@github.com> đã viết:

**Comment by STEW1327** [comment]
can you send plex to my now tv box

**Pull request merged**